website/docs/d/pi_instance.html.markdown

```diff
@@ -20,58 +18,59 @@ data "ibm_pi_instance" "ds_instance" {
 
 - `deployment_type` - (String) The custom deployment type.
 - `health_status` - (String) The health of the instance.
 - `id` - (String) The unique identifier of the instance.
-- `license_repository_capacity` - The VTL license repository capacity TB value. Only available with VTL instances.
-- `memory` - (Float) The amount of memory that is allocated to the instance.
-- `minproc`- (Float) The minimum number of processors that must be allocated to the instance. 
+- `license_repository_capacity` - (Integer) The VTL license repository capacity TB value. Only available with VTL instances.
+- `maxmem`- (Float) The maximum amount of memory that can be allocated to the instance without shutting down or rebooting the `LPAR`.
 - `maxproc`- (Float) The maximum number of processors that can be allocated to the instance without shutting down or rebooting the `LPAR`.
-- `max_virtual_cores` - (Integer) The maximum value that you increase without a reboot.
+- `max_virtual_cores` - (Integer) The maximum number of virtual cores that can be assigned without rebooting the instance.
+- `memory` - (Float) The amount of memory that is allocated to the instance.
 - `minmem`- (Float) The minimum amount of memory that must be allocated to the instance.
-- `maxmem`- (Float) The maximum amount of memory that can be allocated to the instance without shutting down or rebooting the `LPAR`.
-- `min_virtual_cores` - (Integer) The minimum cores assigned to an instance.
-- `networks` - List of objects - The network associated with this instance.
+- `minproc`- (Float) The minimum number of processors that must be allocated to the instance. 
+- `min_virtual_cores` - (Integer) The minimum number of virtual cores that can be assigned without rebooting the instance.
+- `networks` - (List) List of networks associated with this instance.
 
   Nested scheme for `networks`:
-  - `ip` - (String) The IP address of the instance.
   - `external_ip` - (String) The external IP address of the instance.
+  - `ip` - (String) The IP address of the instance.
   - `macaddress` - (String) The MAC address of the instance.
   - `network_id` - (String) The network ID of the instance.
   - `network_name` - (String) The network name of the instance.
   - `type` - (String) The type of the network.
+
+- `pin_policy` - (String) The pinning policy of the instance.
 - `placement_group_id`- (String) The ID of the placement group that the instance is a member.
 - `processors` - (Float) The number of processors that are allocated to the instance.
 - `proctype` - (String) The procurement type of the instance. Supported values are `shared` and `dedicated`.
 - `shared_processor_pool`- (String) The name of the shared processor pool for the instance.
 - `shared_processor_pool_id` - (String)  The ID of the shared processor pool for the instance.
 - `status` - (String) The status of the instance.
+- `storage_connection` - (String) - The storage connection type for the instance.
```

Comment:
there's an extra dash between (String) and The

Reply:
Updated